ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -55,6 +55,48 @@ code change.
      - Bare-metal
      - \-
 
+   * - :ref:`aarch64-unknown-nto-qnx710`
+     - ``aarch64-unknown-nto-qnx710``
+     - Cross-compilation
+     - Full
+     - Can only be cross-compiled from :ref:`x86_64-unknown-linux-gnu` and :ref:`x86_64-pc-windows-msvc`.
+
+   * - :ref:`x86_64-pc-nto-qnx710`
+     - ``x86_64-pc-nto-qnx710``
+     - Cross-compilation
+     - Full
+     - Can only be cross-compiled from :ref:`x86_64-unknown-linux-gnu` and :ref:`x86_64-pc-windows-msvc`.
+
+Quality managed targets
```

Comment:
This should not have been included.24.11 does not include a Mac QM: https://github.com/ferrocene/ferrocene/blob/0986e99a35e79092a3773ef5c847301a8bffdce3/.circleci/workflows.yml#L478-L566